azkaban-web-server/src/main/java/azkaban/scheduler/QuartzScheduler.java

```diff
@@ -45,7 +45,7 @@
 public class QuartzScheduler {
 
   //Unless specified, all Quartz jobs's identities comes with the default job name.
-  private static final String DEFAULT_JOB_NAME = "job1";
+  public static final String DEFAULT_JOB_NAME = "job1";
```

Comment:
This should not be called Flow Trigger. FlowTrigger should has its own flowTrigger job name, for instance, "flowtrigger"

Reply:
we can do it in next PR.